app/forms/nsm/youth_court_fee_form.rb

```diff
@@ -0,0 +1,48 @@
+module Nsm
+  class YouthCourtFeeForm < BaseAdjustmentForm
+    COMMENT_FIELD = 'youth_court_fee_adjustment_comment'.freeze
+    LINKED_CLASS = V1::AdditionalFee
+
+    attribute :remove_youth_court_fee
+    attribute :youth_court_fee_adjustment_comment
+
+    def save
+      return false unless valid?
+
+      remove_bool = ActiveModel::Type::Boolean.new.cast(remove_youth_court_fee)
+
+      process_field(value: !remove_bool, field: 'include_youth_court_fee')
+
+      true
+    end
+
+    private
+
+    def ensure_original_field_value_set(field)
+      if selected_record["#{field}_original"].present?
+        selected_record.delete("#{field}_original")
+        selected_record.delete(self.class::COMMENT_FIELD)
+      else
+        selected_record["#{field}_original"] ||= selected_record[field]
+      end
+    end
+
+    def explanation_required?
+      remove_youth_court_fee == 'true'
```

Comment:
I'm not convinced by this. In every other form we require an explanation any time the record is changed from its previous state, even if it's being reverted to its original state. That might be the system behaving wrong, but if it is wrong we should fix it everywhere. If you really want to buck the trend here and only here, it'd be worth adding a comment explaining why this is a special case with a different UX.